src/main/java/com/powsybl/openloadflow/ac/solver/NewtonRaphson.java

```diff
@@ -85,8 +85,8 @@ public void reportAndLogLargestMismatchByAcEquationType(ReportNode reportNode, E
                         double busPhi = equationSystem.getStateVector().get(busPhiRow);
                         LfBus bus = network.getBus(elementNum);
                         double busNominalV = bus.getNominalV();
-                        double busSumP = bus.getP().eval() * PerUnit.SB;
-                        double busSumQ = bus.getQ().eval() * PerUnit.SB;
+                        double busInjectionP = bus.getTargetP() * PerUnit.SB;
+                        double busInjectionQ = bus.getTargetQ() * PerUnit.SB;
 
```

Comment:
@caioluke not sure why you call this quantity a busInjectionQ. For a PV node the targetQ is irrelevant.
You may also miss some reactive terms that are usually considered as bus injection like shunt and static var compensator.

Reply:
@vidaldid-rte what about reporting the targetQ for PQ buses and bus.getQ().eval() for PV buses?
How would I get all these other reactive terms?

Reply:
IM' not sure what is expected  in the report and if it is really possible do derive a bus injection before the NR converges.

For something named bus injection, I would understand if as all the injections that do not come from the branches. 
I would define it as follow for a Q or P equation
rhs - sum of all active equation terms in the equation for type ElementType.BRANCH
although it has no physical meaning before the NR converges because it includes the equation mismatch
The above expression would include
    The equation mismatch:
    For P (if inactive) , the slack at slack bus
    For Q, if PV node and Q equatin inactive,  the reactive power to distribute to the generators controlling tension
    The shunt terms
    The HVDC station terms (AC emulation mode)
    The static injection (targetQ ...)

Reply:
OK. I agree that the report is misleading because it labels the quantity "bus injection".
It is hard to give a user friendly name for that quantity. In the context of Newton Raphson it is the Left Hand Side of the equation (the computed part) that is supposed when the equation is active to match the static part, the right hand side.